SQF/dayz_code/stringtable.xml

```diff
@@ -1310,11 +1310,11 @@
 			<Dutch>Deze %1 heeft niet voldoende brandstof.</Dutch>
 		</Key>
 		<Key ID="str_fill_notenough">
-			<English>%1 does not have enough fuel to fill jerry can(s). (Estimated %2 Liters left)</English>
+			<English>%1 does not have enough fuel. Estimated %2 litres left (You need %3 litres)</English>
 			<German>%1 hat nicht genug Treibstoff, um einen Benzinkanister komplett zu füllen.(Ungefähr %2 Liter übrig)</German>
```

Comment:
For the german string:

%1 hat nicht genug Kraftstoff, um einen Kraftstoffbehälter komplett zu füllen. Es sind noch ungefähr %2 Liter übrig. (Du brauchst %3 Liter)